src/main/java/uk/gov/hmcts/ccd/endpoint/exceptions/RestExceptionHandler.java

```diff
@@ -50,6 +60,29 @@ public RestExceptionHandler(AppInsights appInsights) {
                 .body(error);
     }
 
+    @ExceptionHandler(CaseValidationException.class)
+    @ResponseBody
+    public ResponseEntity<HttpError> handleCaseValidationException(HttpServletRequest request, CaseValidationException exception) {
+
+        // read field IDs from CaseValidationException.details (i.e. avoid using full error details as this may contain user data)
+        List<String> fieldIds = new ArrayList<>();
+        if (exception.getDetails() != null) {
+            String detailsJson = this.objectMapperService.convertObjectToString(exception.getDetails());
+            fieldIds = JsonPath.read(detailsJson, "$..field_errors[*].id");
+        }
+
+        Map<String, String> customProperties = new HashMap<>();
+        customProperties.put("CaseValidationError field IDs", Arrays.toString(fieldIds.toArray()));
+
+        LOG.warn("{}: The following list of fields are in an error state: {}", exception.getMessage(), fieldIds, exception);
```

Comment:
```suggestion
        LOG.warn("{}: The following list of fields are in an invalid state: {}", exception.getMessage(), fieldIds, exception);
```

Reply:
Thanks that sounds much cleaner. Updated in next push.